client/my-sites/media-library/scale.jsx

```diff
@@ -0,0 +1,139 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import { debounce, partial } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormRange from 'components/forms/range';
+import Gridicon from 'components/gridicon';
+import SegmentedControl from 'components/segmented-control';
+import SegmentedControlItem from 'components/segmented-control/item';
+import { setPreference, savePreference } from 'state/preferences/actions';
+import { getPreference } from 'state/preferences/selectors';
+
+/**
+ * Constants
+ */
+const SLIDER_STEPS = 100;
+const SCALE_TOUCH_GRID = 0.32;
+const SCALE_CHOICES = [ 0.077, 0.115, 0.157, 0.24, 0.323 ];
```

Comment:
This seems a bit "magical" on its own - I would add a comment similar to what was there before:

```
// Choices are 12, 8, 6, 4, and 3 items per row, with some horizontal padding between items
```